onc/+onc/OncDelivery.m

```diff
@@ -104,7 +104,7 @@
             %
             % Documentation: https://wiki.oceannetworks.ca/display/CLIBS/Data+product+download+methods
             
-            if ~exist('waitComplete','var'), waitComplete = false; end
+            if ~exist('waitComplete','var'), waitComplete = true; end
             url = sprintf('%sapi/dataProductDelivery', this.baseUrl);
             log = onc.DPLogger();
```

Comment:
This is actually for line 101: you can remove the TODO. It seems strange that I cannot comment on unedited lines of code in a review. Ah I'm not the only one that finds that strange: https://github.com/orgs/community/discussions/4452